packages/builder/src/types.ts

```diff
@@ -7,55 +7,37 @@ import * as types from "./internal/core/params/argument-types";
 
 // IMPORTANT: This t.types MUST be kept in sync with the actual types.
 
-interface CommonNetworkConfig {
-  //chainId?: number;
-  //from?: string;
-  //gas?: "auto" | number;
-  //gasPrice?: "auto" | number;
-  //gasMultiplier?: number;
-}
-
-interface BuilderNetworkAccount {
-  privateKey: string;
-  balance: string;
-}
-
-export interface BuilderNetworkConfig extends CommonNetworkConfig {
-  accounts?: BuilderNetworkAccount[];
-  //blockGasLimit?: number;
-  //hardfork?: string;
-  throwOnTransactionFailures?: boolean;
-  throwOnCallFailures?: boolean;
-  //loggingEnabled?: boolean;
-  //allowUnlimitedContractSize?: boolean;
-  //initialDate?: string;
-}
-
 export interface HDAccountsConfig {
   mnemonic: string;
   initialIndex?: number;
   count?: number;
   path?: string;
 }
 
-export interface OtherAccountsConfig {
-  type: string;
+export type NetworkAccounts =
+  | string[]
+  | HDAccountsConfig;
+
+interface CommonNetworkConfig {
+  accounts?: NetworkAccounts;
+  chainName?: string;
+  //from?: string;
 }
 
-export type NetworkConfigAccounts =
-  | "remote"
-  | string[]
-  | HDAccountsConfig
-  | OtherAccountsConfig;
+export interface AlgoDevChainConfig extends CommonNetworkConfig {
```

Comment:
Will we have non-Algo and non-Dev configs?

Reply:
maybe - this is for milestone 7 (optional)
> Lightweight blockchain (simulator) to quickly run and test dapps and smart-contracts without blockchain overhead (network / consensus). This will be a stripped Algorand blockchain written in Go using the Algorand codebase.